pkg/network/node/ovscontroller.go

```diff
@@ -480,6 +563,9 @@ func (oc *ovsController) UpdateEgressNetworkPolicyRules(policies []networkapi.Eg
 }
 
 func (oc *ovsController) AddHostSubnetRules(subnet *networkapi.HostSubnet) error {
+	defer oc.updateOVSMetrics()
+	defer oc.updateARPMetrics()
```

Comment:
The ARP table size is not going to change as a result of adding or removing node-related OVS flows. It only changes with *actual* traffic, not potential traffic. You really need to update that on a timer or something.